portality/models/v2/application.py

```diff
@@ -210,11 +206,8 @@ def save(self, sync_owner=True, **kwargs):
             self.set_id(self.makeid())
 
         if self.application_type == constants.APPLICATION_TYPE_UPDATE_REQUEST:
-            rc = redis.Redis(host=app.config.get("HUEY_REDIS_HOST"), port=app.config.get("HUEY_REDIS_PORT"))
-            aid = rc.get(self.current_journal)
-            if aid is not None and aid != self.id:
-                raise ConcurrentUpdateRequestException()
-            rc.set(self.current_journal, self.id, ex=10)
+            cs = ConcurrencyPreventionService()
```

Comment:
Service should be loaded via the `DOAJ` entry point, same way as the other services

Reply:
Why do we do this?

Reply:
It follows a model from more strongly typed languages where the `DOAJ` entry point is effectively a factory for the services, allowing them to (theoretically) be swapped out for other implementations (e.g. mocked ones for testing).  In practice we haven't actually used it that way, so there's an argument for refactoring.  But: if you want to propose a change to the way we do it, it has to be done everywhere, what I don't want is a hybrid set of patterns for this.